standards/git-commands.md

```diff
@@ -0,0 +1,34 @@
+# Git Commands
+Git is a distributed version-control system for tracking changes in source code during software development. It is designed for coordinating work among programmers, but it can be used to track changes in any set of files. Its goals include speed, data integrity, and support for distributed, non-linear workflows.
+
+At Shift3, we use git for a majority of our software projects.
+
+## Shift3 git specific resources
+1. [Commits](commits.md)
+2. [Branching](branching.md)
+3. [Code Versioning](code-versioning.md)
+
+### Lesser known git commands
+
+#### Git Stash
+
+Git Official Documentation: https://git-scm.com/docs/git-stash
+
+Git Stash is used to record the current state of the working directory and index (your code within your current branch) but want to go back to a clean working directory. In order to "stash" your code away for future use, first `git add` the files you want to stash away but *do not use git commit*. You use the `git stash push` command to create your stash. You can switch your branches and then apply the same code to the new branch with `git stash pop`. 
+
+In the below example, we are pushing this readme into our `foo-999-big-feature` branch which leaves us with a clean working directory/index within our origional branch `mac-192-lesser-known-git`.
```

Comment:
Spelling of `original`

Reply:
Doh!

Should be fixed via: dd8bf571fbb14c9765797c3c712d90912ab3cc3a